test/bravery-components/braveryPanelTest.js

```diff
@@ -954,26 +954,23 @@ describe('Bravery Panel', function () {
     })
   })
 
-  // TODO: Fix iframe tests (See: #8760)
   describe('Adblock stats iframe tests', function () {
     Brave.beforeEach(this)
     beforeEach(function * () {
       yield setup(this.app.client)
-      yield this.app.client
-        .waitForDataFile('adblock')
```

Comment:
I thought it was but removing it increased test speed and adblock was recognized same way. For some reason dataFile is unreachable for a long time and was leading to intermittent fail. Same for other dataFile requests but for adblock seems worse.